layout/boxlayout.go

```diff
@@ -107,35 +107,32 @@ func (g *boxLayout) Layout(objects []fyne.CanvasObject, size fyne.Size) {
 // For a BoxLayout this is the width of the widest item and the height is
 // the sum of of all children combined with padding between each.
 func (g *boxLayout) MinSize(objects []fyne.CanvasObject) fyne.Size {
-	spacerCount := 0
 	minSize := fyne.NewSize(0, 0)
-	added := false
+	addPadding := false
```

Comment:
I think it would be cleaner to remove this flag and just do:
```suggestion
	for i, child := range object {
```
```suggestion
	if i != 0 {
```

Reply:
This won’t work because the first element might be ignored (invisible or a spacer).